src/GlobalLoop.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace React\EventLoop;
+
+final class GlobalLoop
+{
+    /**
+     * @internal
+     *
+     * @var LoopInterface
+     */
+    public static $loop;
```

Comment:
@jsor @WyriHaximus This is one reason why https://github.com/async-interop/event-loop uses a static class instead of functions and has the clear `Loop::execute` scoping. The current loop can be a private property and swapping out the loop is very easy and not something you have to worry about.